src/rgw/rgw_admin.cc

```diff
@@ -6211,6 +6275,17 @@ int main(int argc, const char **argv)
     }
   }
 
+  if (opt_cmd == OPT_RESHARD_CLEANUP) {
+    if (bucket_name.empty()) {
```

Comment:
probably makes sense to support `--bucket-id` as an argument here to start from an instance other than `orig_bucket.marker`